flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/container/FlinkContainers.java

```diff
@@ -254,6 +259,55 @@ public void submitSQLJob(SQLJobSubmission job) throws IOException, InterruptedEx
         }
     }
 
+    /**
+     * Submits the given job to the cluster.
+     *
+     * @param job job to submit
+     */
+    public JobID submitJob(JobSubmission job) throws IOException, InterruptedException {
```

Comment:
`submitJobWithCLI`? I could imagine that someone wants to specificially test different entry points.

Reply:
I have no hard feelings about the name of this method but I wanted to keep it consistent with the `ClusterController`[1] that is used for the current e2e tests. I thought it maybe makes the migration easier because developers are used to that name.

[1] https://github.com/apache/flink/blob/master/flink-end-to-end-tests/flink-end-to-end-tests-common/src/main/java/org/apache/flink/tests/util/flink/ClusterController.java#L37